dnf-behave-tests/features/builddep.feature

```diff
@@ -80,3 +80,62 @@ Scenario: Builddep on SPEC with non-available Source0
 
    Error: Some packages could not be found.
    """
+
+@bz1758459
+Scenario: I can exclude package version and install the best candidate
+  Given I use repository "dnf-ci-fedora"
+    And I enable plugin "builddep"
+  Given I create file "dummy-pkg.spec" with
+   """
+   Name: dummy-pkg
+   Summary: dummy-pkg summary
+   Version: 1.0
+   Release: 1
+   License: GPL
+   BuildRequires: nodejs
+   %description
+   This is a dummy-pkg description
+   %build
+   %files
+   %changelog
+   """
+   When I execute dnf with args "builddep {context.dnf.installroot}/dummy-pkg.spec --exclude nodejs-1:5.12.1-1.fc29 --best"
```

Comment:
Unless I'm mistaken this test isn't related to modularity and so you don't need the `nodejs` package. To avoid adding another version of the `nodejs` package to the `dnf-ci-fedora` repository, could you use a different package that already has multiple versions in the testing repositories? (there are multiple such packages in `dnf-ci-fedora` that have another version in `dnf-ci-fedora-updates`)